From: Shraddha Barke <shraddha.6596@gmail.com>
Date: Sun, 18 Oct 2015 13:55:38 +0530
Subject: [PATCH] libceph: remove con argument in handle_reply()
Git-commit: 70cf052d0c4b60b6fbb981380660893306b9f172
Patch-mainline: v4.4
References: Fate#318586


Since handle_reply() does not use its con argument, remove it.

Signed-off-by: Shraddha Barke <shraddha.6596@gmail.com>
Signed-off-by: Ilya Dryomov <idryomov@gmail.com>
Acked-by: NeilBrown <neilb@suse.com>

---
 net/ceph/osd_client.c |    5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

--- a/net/ceph/osd_client.c
+++ b/net/ceph/osd_client.c
@@ -2092,8 +2092,7 @@ static void complete_request(struct ceph
  * handle osd op reply.  either call the callback if it is specified,
  * or do the completion to wake up the waiting thread.
  */
-static void handle_reply(struct ceph_osd_client *osdc, struct ceph_msg *msg,
-			 struct ceph_connection *con)
+static void handle_reply(struct ceph_osd_client *osdc, struct ceph_msg *msg)
 {
 	void *p, *end;
 	struct ceph_osd_request *req;
@@ -3248,7 +3247,7 @@ static void dispatch(struct ceph_connect
 		ceph_osdc_handle_map(osdc, msg);
 		break;
 	case CEPH_MSG_OSD_OPREPLY:
-		handle_reply(osdc, msg, con);
+		handle_reply(osdc, msg);
 		break;
 	case CEPH_MSG_WATCH_NOTIFY:
 		handle_watch_notify(osdc, msg);
